set-balance.md

```diff
@@ -329,13 +369,8 @@ Helpers for calling `set_free_balance` and `set_reserved_balance`.
 
     rule [balance-set-reserved]:
          <k> set_balance_reserved(WHO, RESERVED_BALANCE') => set_reserved_balance(WHO, RESERVED_BALANCE') ... </k>
-         <totalIssuance> ISSUANCE => ISSUANCE +Int (RESERVED_BALANCE' -Int RESERVED_BALANCE) </totalIssuance>
-         <account>
-           <accountID> WHO </accountID>
-           <reservedBalance> RESERVED_BALANCE </reservedBalance>
-           ...
-         </account>
-      requires #inWidth(64, ISSUANCE +Int (RESERVED_BALANCE' -Int RESERVED_BALANCE))
+         <totalIssuance> ISSUANCE => ISSUANCE +Int (RESERVED_BALANCE' -Int reserved_balance(WHO)) </totalIssuance>
+      requires #inWidth(64, ISSUANCE +Int (RESERVED_BALANCE' -Int reserved_balance(WHO)))
```

Comment:
```suggestion
      requires #inWidth(96, ISSUANCE +Int (RESERVED_BALANCE' -Int reserved_balance(WHO)))
```

Polkadot uses u128 for balances.

Reply:
Should this be `128` then instead? Where did `96` come from?